src/workflows-app/utils/job-common.js

```diff
@@ -0,0 +1,126 @@
+import { isEmpty, isNil, kebabCase } from 'lodash';
```

Comment:
I think this file may be somewhat a duplicate of `src/workflows-app/components/job-common`